manifests/profiles/external.yaml

```diff
@@ -5,7 +5,7 @@ kind: IstioOperator
 spec:
   components:
     base:
-      enabled: false
+      enabled: true
```

Comment:
This is not needed. The `external` profile will install the CRDs if `values.global.configCluster` is `true`.